src/app/centers/createcenter/createcenter.component.scss

```diff
@@ -0,0 +1,12 @@
+.container {
+    max-width1: 37rem;
+  }
+
+  .example-full {
```

Comment:
Remove these unnecessary classes. Use only when they are really necessary and name them properly.